### DISCUSSION TITLE
Question about vendor js

### DISCUSSION THREAD

**Discussion by bogdanpetrea** [bhch/django-jsonform#105] [General]
Hi,

I've been inspecting the vendor files from the static directory, and I found that [react-modal.min.js](https://github.com/bhch/django-jsonform/blob/master/django_jsonform/static/django_jsonform/vendor/react-modal.min.js) has a few differences from the minified version found at https://registry.npmjs.org/react-modal/-/react-modal-3.15.1.tgz (as mentioned in [package-lock.json](https://github.com/bhch/react-json-form/blob/master/package-lock.json#L9043)).

`react-dom` and `react` seemed to match with their online versions, just not `react-modal`.

Any clue as to why and can it fixed?

**Comment by bhch** [comment]
The Django package includes an older version ([v3.14.3](https://github.com/reactjs/react-modal/commit/fd193f2a66b3ad54b4cecfe1c2d3bb5d1475ae6b)) of react-modal.

When I release a new version of django-jsonform, I have to manually copy/paste the JS files from the development repository (react-json-form). I must have forgot update the react-modal lib.

**Reply by bogdanpetrea** [reply]
Thanks! I compared to [3.14.3](https://cdnjs.cloudflare.com/ajax/libs/react-modal/3.14.3/react-modal.min.js) (and a few other versions) before posting here, and they still didn't match.

**Reply by bhch** [reply]
I'm seeing no differences: https://www.diffchecker.com/T7J33xLu

Could you check again?

**Reply by bogdanpetrea** [reply]
Hmm you're right.
I think what happened is I checked locally with `git diff` and pasting the online version added a newline or something like that. Might be my editor automatically adding a newline after a set number of characters, or I don't know.

Even after I pasted from npm in diffchecker I got a weird newline https://www.diffchecker.com/7MNzHUCv/ , but pasting from cloudflare shows no difference there indeed.

I didn't look closer for the other versions at what the differences were, as initially for 3.15.1 they were obvious...

So my bad in the end, sorry for wasting your time.
